docs/guides/platform/CheCluster_crd_v1.md

```diff
@@ -45,4 +47,49 @@
 |status.openShiftOAuthUserCredentialsSecret|string|OpenShift OAuth secret in `openshift-config` namespace that contains user credentials for HTPasswd identity provider|
 |status.openShiftoAuthProvisioned|boolean|Indicates whether an Identity Provider instance, Keycloak or RH-SSO, has been configured to integrate with the OpenShift OAuth|
 |status.pluginRegistryURL|string|Public URL to the plugin registry|
-|status.reason|string|A brief CamelCase message indicating details about why the Pod is in this state|
\ No newline at end of file
+|status.reason|string|A brief CamelCase message indicating details about why the Pod is in this state|
+
+
+
+
+
+
+
+
+
+## <span translate="no">CheCluster</span>/v2
+
+<span translate="no">CheCluster</span> is the schema for the <span translate="no">checlusters</span> API.
+
+### spec
+
+**Description:** Defines the desired state of CheCluster.
+
+**Type:**  object
+
+| Property            | Type        | Description                       |
+|---------------------|-------------|-----------------------------------|
+|spec                 |object       |Defines the desired state of CheCluster|
+|spec.components|object|Che components configuration.|
+|spec.containerRegistry|object|Configuration of an alternative registry that stores Che images.|
+|spec.devEnvironments|object|Development environment default configuration options.|
+|spec.gitServices|object|A configuration that allows users to work with remote Git repositories.|
+|spec.networking|object|Networking, Che authentication, and TLS configuration.|
+
+### status
+
+**Description:** Defines the observed state of CheCluster.
+
+**Type:**  object
+
+| Property            | Type        | Description                       |
+|---------------------|-------------|-----------------------------------|
+|status.chePhase|string|Specifies the current phase of the Che deployment.|
+|status.chePhase|string|Public URL of the Che server.|
```

Comment:
Are all the properties in this table supposed to have the same property name `status.chePhase`?

Reply:
Good catch. The property names are different. Changes are made in new commit.